index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="index.css" />
   </head>
   <body>
```

Comment:
- I love how organized you are with your codes 💯 👍 
- [ ] However, I think you still have a few things to check because I am unable to check the functionality of your code as the navigation links are not working. Kindly check the implementation of your single-page app, and include appropriate reference links in the `href` attributes of your navigation.

This will improve access to your app, and ensures nice user experience when navigation seems responsive 💪 

_This is all I see at the moment...can't access the form to add new books nor contact you for any opportunity in case I have one 😄 _ 👇 

![image](https://user-images.githubusercontent.com/99715640/220326460-8678b601-f420-4f8d-861b-7bea22fa8dec.png)

Reply:
:clap::clap::clap: on fixing this issue :blush: